### ISSUE TITLE
Add license

### ISSUE THREAD

**Issue by scoopgracie** [qufighter/ColorPick#4]
Would you mind adding a license?

**Comment by qufighter** [comment]
I think this is a great suggestion!  The reason I never added on is I have basically the most protection under the law without adding one.  Any license I add is going to be more liberal.  

I have some ideas about it... and I have a vision of what I really want the license to be... but I also don't want it to be something I ever have to go into court to enforce either... so I'm not sure I can really go as liberal as my vision for it.  I don't want to relinquish any control and at the same time, managing a single webstore listing (or two) is really a hassle and I don't want to do it.  The webstore these days is nightmare of inability to fix any bug without long turn around times.  What a business idea to manage multiple webstore listings for someone.... nasty idea.... might be relevant someday unfortunately. 

You  know how many people want to buy this just to put spyware in it?  I still haven't written the clause in this license to fully address that but it is on it's way...

I will share what I have drafted though, maybe it's what you wanted, maybe not.

https://github.com/qufighter/ColorPick/compare/next

It's not the license though, it's just a draft.  Even when it's merged I still think it's just a draft.  I haven't gone to law school yet and I'm not paying for it.  The basic idea is existing licenses are great and have a lot of sensible things in them, but at the core this extension intends to break even someday, it's not for theft, it's for sale.

That being said, if you just want to use one file, or part of a file, licensing that is different... I'd break the code out into it's own repo, add a license there, etc (not that you couldn't use what I wrote, it'd just be less complicated to facilitate the use of a more relevant license for your needs).

Thanks for the suggestion!
